backend/api/Areas/Admin/Mapping/Parcel/BuildingMap.cs

```diff
@@ -42,12 +43,10 @@ public void Register(TypeAdapterConfig config)
             config.NewConfig<Model.BuildingModel, Entity.Building>()
                 .IgnoreNonMapped(true)
                 .Map(dest => dest.Id, src => src.Id)
-                .Map(dest => dest.LocalId, src => src.LocalId)
                 .Map(dest => dest.AgencyId, src => src.AgencyId)
                 .Map(dest => dest.Name, src => src.Name)
                 .Map(dest => dest.Description, src => src.Description)
-                .Map(dest => dest.Latitude, src => src.Latitude)
-                .Map(dest => dest.Longitude, src => src.Longitude)
+                .Map(dest => dest.Location, src => new NetTopologySuite.Geometries.Point(src.Longitude, src.Latitude) { SRID = 4326 })
```

Comment:
what is this magic number?

Reply:
A spatial reference identifier (SRID) is a unique identifier associated with a specific coordinate system, tolerance, and resolution.  4326 is a number used for geographical coordinates.  I don't know more than this.  I'm going to add a commit to centralize the creation of points.